salt/modules/event.py

```diff
@@ -108,15 +110,20 @@ def fire(data, tag):
 
         salt '*' event.fire '{"data":"my event data"}' 'tag'
     """
+    if timeout is None:
+        timeout = 60
+    timeout = timeout * 1000
     try:
-        with salt.utils.event.get_event(
-            "minion",  # was __opts__['id']
+        event = salt.utils.event.get_event(
```

Comment:
we should still use `with` here to ensure everything is closed properly with salt's event class.